llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -869,7 +840,10 @@ void VPIRInstruction::execute(VPTransformState &State) {
     State.Builder.SetInsertPoint(PredBB, PredBB->getFirstNonPHIIt());
     Value *V = State.get(ExitValue, VPLane(Lane));
     auto *Phi = cast<PHINode>(&I);
-    Phi->addIncoming(V, PredBB);
+    if (Phi->getBasicBlockIndex(PredBB) == -1)
```

Comment:
Worth commenting about the two types of VPIRBB's - those that add a new ("extra") incoming and those that update an existing one?

Reply:
Updated to add a comment, thanks!